.kokoro/psm_interop_kokoro_lib.sh

```diff
@@ -158,6 +158,51 @@ psm::security::run_test() {
   psm::tools::run_verbose python -m "tests.${test_name}" "${PSM_TEST_FLAGS[@]}"
 }
 
+# --- DualStack TESTS ------------------
+
+#######################################
+# DualStack Test Suite setup.
+# Outputs:
+#   Prints activated cluster names.
+#######################################
+psm::dual_stack::setup() {
```

Comment:
Here and elsewhere: rename `dual_stack` to `dualstack`.
This is to be consistent with how we already refer to this functionality in the framework (e.g. `--enable_dualstack` flag).